problemsets-mat223/modules/module9.tex

```diff
@@ -71,7 +71,29 @@
 	\]
 	For each of $S$ and $T$, determine whether the transformation is linear.
 
-	XXX Finish
+	For $\mathcal T$:
+	
+	Let vectors $\vec u=\mat{u_1\\u_2},\vec v=\mat{v_1\\v_2}$ and scalar $\alpha$ be given. So $\vec u +\vec v=\mat{u_1+v_1\\u_2+v_2}$ and $a\vec u = \mat{a u_1\\a u_2}$. We then have 
```

Comment:
`$\vec u=\mat{u_1\\u_2},\vec v=\mat{v_1\\v_2}$` -> `$\vec u=\mat{u_1\\u_2}$, $\vec v=\mat{v_1\\v_2}$`

`and` -> `, and let $\alpha$ be a scalar.`

Reply:
Remove the second sentence and replace it with "We need to verify that $T(\vec u+\vec v)=T\vec u+T\vec v$ and $...$. Computing, we see"